### PR TITLE
Ported tests excluding operator tests

### PR DESCRIPTION
**Pull request by badochov**
Didn't know what to do so did this. Out of scope of iteration 1

### CODE REVIEW

**Comment by jmolinski** [comment]
Can it wait a bit for merging? I'd like to have the runtime ported before

**Comment by badochov** [comment]
Yeah it can as I said it's not scheduled for first iteration I was just bored

File: src/environment/tests.go [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+		return int64(p), ok
+	}
+	if p, ok := v.(uint16); ok {
+		return int64(p), ok
+	}
+	if p, ok := v.(uint32); ok {
+		return int64(p), ok
+	}
+	if p, ok := v.(uint64); ok {
+		return int64(p), ok
```
Comment (jmolinski) [inline src/environment/tests.go:48]: uint64 can be out of scope for int64; shouldn't we check for this case?

**Review by jmolinski — APPROVED** [review]
Great work, looks good to me

We'll need tests for these functions, maybe add an issue?

File: src/environment/tests.go (hunk 1 above)
Comment (badochov) [reply src/environment/tests.go:48]: We have to draw a line somewhere on size of the ints, this way it's easier and honestly who would use 64 uint in a template.  In this case including uint64 would not complicate the code too much but in operator the code would get much more complex I don't think it's worth it at this stage of the project.

**Comment by badochov** [comment]
> Great work, looks good to me
> 
> We'll need tests for these functions, maybe add an issue?

I added tests

File: src/environment/tests.go (hunk 1 above)
Comment (jmolinski) [reply src/environment/tests.go:48]: > who would use 64 uint in a template

we have to remember that python supports bignums, so somebody for sure used that

> In this case including uint64 would not complicate the code too much

I just thought about doing

```
if uintValue64 out of int64 scope {
  return 0, error("value out of scope for int64")
}
``` 
and cascade the error down 

----

> but in operator the code would get much more complex 

ok i get the point, let's leave it as it is

**Review by jmolinski — APPROVED** [review]
Great!

**Pull request merged**